src/bom/bitcode.rs

```diff
@@ -299,13 +309,16 @@ fn build_bitcode_arguments(chan : &mut mpsc::Sender<Option<Event>>,
             // Reject arguments matching any of the user-provided regexes.  Note
             // that this is of course as unsafe as users make it.  In
             // particular, rejecting '-o' would be very bad.
-            skip_next = clang_support::next_arg_is_option_value(arg);  // hopeful here...
+            skip_next = clang_support::next_arg_is_option_value(arg);
             continue;
         } else {
-            if arg.to_str().unwrap().starts_with("-o") {
-                modified_args.push("-o".into());
-            } else {
-                modified_args.push(OsString::from(arg.to_owned()));
+            if ! arg.to_str().unwrap().starts_with("-o") {
+                if arg.to_str().unwrap().starts_with("-") {
```

Comment:
Maybe this is better for a separate PR, but we probably want to make this into a `to_string_lossy` (which doesn't require an `unwrap`)